docs/CHANGELOG.asciidoc

```diff
@@ -85,6 +85,48 @@ include::upgrade-notes.asciidoc[]
 
 For information about the {kib} 8.16.0 release, review the following information.
 
+The 8.16.0 release includes the following known issues.
+
+[float]
+[[known-issues-8.16.0]]
+=== Known issues
+
+[discrete]
+[[known-199902]]
+.Stack Monitoring shows "Unable to load page error"
+[%collapsible]
+====
+*Details* +
+The Stack Monitoring pages Overview, Nodes, Logs can stop working with the error "Unable to load page error". The Stack trace mentions `TypeError: Cannot read properties of undefined (reading 'logsLocator')`.
+
+*Workaround* +
+Disabling the `Set feature visibility > Logs` feature at Kibana Space settings level will prevent the error to occur. Please note the `Logs` feature will not be available on such space.
+
+It's also possible to `Observability > Logs` feature privilege to `None` on the role level - this will hide the `Logs` feature for individual users and prevent the error for these users as well.
```

Comment:
```suggestion
It's also possible to set the `Observability > Logs` feature privilege to `None` on the role level. This will hide the `Logs` feature for individual users and prevent the error for these users as well.
```